src/distribution/gaussian.rs

```diff
@@ -414,10 +445,26 @@ mod tests {
     use assert;
     use prelude::*;
 
+    use crate::distribution::FromIter;
+
     macro_rules! new(
         ($mu:expr, $sigma:expr) => (Gaussian::new($mu, $sigma));
     );
 
+    #[test]
+    fn iter() {
+        // Create a sample distribution to create an iterator from
+        let mut source = source::default(42);
+        let distribution = new!(1.0, 2.0);
+        let sampler = Independent(&distribution, &mut source);
+        let samples = sampler.take(10000).collect::<Vec<_>>();
+        let derived_distribution = Gaussian::from_iter(samples.into_iter());
+
+        assert::close(derived_distribution.mu, distribution.mu, 0.1);
```

Comment:
The last one is epsilon. It is way to large. Can you please set it to, say, `1e-8`?

Reply:
There's a reason it's so large.  If you want to have a small epsilon in this function, you'll need to make a very, very large set of random data to exactly match.  1e-8 could require a billion samples or more.  I tried generating with 100,000,000 samples (which makes the test take a few seconds) and only got this close:

`'1.0000279833901966 !~ 1.0'`

Reply:
Ah, I see. Makes sense.